libraries/Microsoft.Bot.Builder.AI.QnA/Helpers/GenerateAnswerHelper.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Helper class for Generate Answer API.
+    /// </summary>
+    internal class GenerateAnswerHelper
+    {
+        private static readonly string QnAMakerName = nameof(QnAMaker);
+        private readonly IBotTelemetryClient telemetryClient;
+        private QnAMakerEndpoint _endpoint;
+        private readonly HttpClient httpClient;
```

Comment:
This should be one of these:
https://docs.microsoft.com/en-us/previous-versions/visualstudio/hh138157(v=vs.118)